docs/dagster-university/next.config.js

```diff
@@ -1,5 +1,5 @@
 const withMarkdoc = require('@markdoc/next.js');
 
-module.exports = withMarkdoc(/* config: https://markdoc.io/docs/nextjs#options */)({
+module.exports = withMarkdoc({schemaPath: './markdoc', mode: 'static'})({
```

Comment:
It's possible that neither are necessary, but it's not really clear to me from the Markdoc documentation what the defaults are.

- `mode: 'static'` renders the markdown pages at build time. This seems correct to me, since as far as I can tell, the contents are static and don't need to do anything special at request time. Should be nice and fast.
- `schemaPath` is intended to point at the `markdoc` directory where we have custom nodes and tags defined. It seems like it should work even without this, but the Markdoc docs include this value, so I added it.